transfers/transfer.py

```diff
@@ -158,8 +178,8 @@ def get_accession_id(dirname):
     script_path = os.path.join(THIS_DIR, 'get-accession-number')
     try:
         p = subprocess.Popen([script_path, dirname], stdin=subprocess.PIPE, stdout=subprocess.PIPE, stderr=subprocess.PIPE)
-    except FileNotFoundError:
-        LOGGER.info('%s does not exist.', script_path)
+    except:
```

Comment:
Should be `except Exception:` so it doesn't catch `KeyboardInterrupt` etc https://docs.python.org/3/library/exceptions.html#exception-hierarchy Any exception handling should specify what exceptions we're expecting to catch, or at a minimum `Exception`.